Userland/Applications/FileManager/main.cpp

```diff
@@ -951,6 +951,10 @@ ErrorOr<int> run_in_windowed_mode(String initial_location, String entry_focused_
     directory_view->set_should_show_dotfiles(show_dotfiles);
     action_show_dotfiles->set_checked(show_dotfiles);
 
+    auto const initial_location_is_hidden = initial_location.contains('.');
```

Comment:
Shouldn't it check if it starts with dot?

Reply:
`initial_location` is the full path to the directory being opened, so checking if it starts with `.` wouldn't suffice.

But you're right that this would show dotfiles if I open a directory named `a.b` as well. Maybe not a huge deal - but would be more correct to instead check if any segment in the `initial_location` starts with `.` instead.